modules/TestUtils.js

```diff
@@ -53,27 +79,29 @@ export const arrayContains = (array, value, compareValues) => {
 }
 
 /**
- * Returns true if the given string contains the value, false otherwise.
- */
-export const stringContains = (string, value) =>
-  string.indexOf(value) !== -1
-
-/**
- * Returns true if the given object is an array.
+ * Returns true if the given object contains the value, false
+ * otherwise. The compareValues function must return false to
+ * indicate a non-match.
  */
-export const isArray = (object) =>
-  Array.isArray(object)
+export const objectContains = (object, value, compareValues) => {
+  if (compareValues == null)
+    compareValues = isEqual
```

Comment:
should this use ES6 default arguments? ie, `compareValue = isEqual` in the signature